cli/paas/install_client.go

```diff
@@ -129,14 +150,46 @@ func (c *InstallClient) showInstallConfiguration(ds *kubernetes.DeploymentSet) {
 	m.Msg("Configuration...")
 }
 
-// deploySet deploys all the deployments in the set, in order.
-func (c *InstallClient) deploySet(ds *kubernetes.DeploymentSet) error {
-	for _, deployment := range ds.Deployments {
-		options := ds.NeededOptions.ForDeployment(deployment.ID())
-		err := deployment.Deploy(c.kubeClient, c.ui, options)
-		if err != nil {
-			return err
+func (c *InstallClient) fillInMissingSystemDomain(domain *kubernetes.InstallationOption) error {
+	if domain.Value.(string) == "" {
+		ip := ""
+		if !c.kubeClient.GetPlatform().HasLoadBalancer() {
+			ips := c.kubeClient.GetPlatform().ExternalIPs()
+			if len(ips) > 0 {
+				ip = ips[0]
+			}
+		} else {
+			c.ui.ProgressNote().Msg("Waiting for LoadBalancer IP on traefik service.")
```

Comment:
Why not a
```
	s := ui.Progress("Waiting for LoadBalancer IP on traefik service.")
	defer s.Stop()
```
to also get the nice dot progress while we are waiting here ?

Reply:
That's the one I was looking for! :D Thanks, changing.